### PR TITLE
Remove player_twi account from target accounts

### PR DESCRIPTION
**Pull request by yusuko**
# Description
This PR is to remove @player_twi from target accounts.
Because this account has too much data and cause memory errors. https://github.com/ookamiinc/twitter-analytics-acquirer/issues/28

So I add next process.

## Note 
If you are eager to get Player_twi account, I recommend that you add one more server.

### CODE REVIEW

File: script.rb [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 DatabaseClient.all.each do |twitter_account|
   next if twitter_account['name'] == 'Playerapp_cbsk'
+  # I remove Player_twi because this account has too much data.
+  # But I want to save cookies data, so deal with it by adding next process.
+  next if twitter_account['name'] == 'Player_twi'
```
Comment (nafu) [inline script.rb:13]: It would be better if we can handle these via environment variable.

Comment (nafu) [reply script.rb:13]: @yusuko ↑

Comment (yusuko) [reply script.rb:13]: Yes, that's true.
Thank you!

Comment (yusuko) [reply script.rb:13]: I create PR to deal with it👍

**Pull request merged**